### PR TITLE
Uncommented Log code

### LINKED ISSUE #160: Logging of spinner data is commented out
We didn't have spinners in the code when the logging was implemented so lines were commented out.  This means we're not logging any spinner data and this needs to be fixed to log correctly.

We also need to ensure we are logging the multi-select dropdown information as well.  as
ID1:ID2:ID3
(colon separated id's)

### PR DESCRIPTION
**Pull request by AdrielStammler**
Some of the logger code was commented, they were from when the weren't dropdowns and they weren't uncommented when converted to dropdowns.

fixes #160

### CODE REVIEW

File: app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java [hunk 1]
```diff
@@ -203,10 +202,10 @@ public void onClick(View view) {
...
                 // Log all of the data from this page
                 Globals.EventLogger.LogData(Constants.LOGKEY_DID_LEAVE_START, String.valueOf(postMatchBinding.checkboxDidLeave.isChecked()));
-//                Globals.EventLogger.LogData(Constants.LOGKEY_CLIMB_POSITION, postMatchBinding.spinnerClimbPosition.getSelectedItem().toString());
-//                Globals.EventLogger.LogData(Constants.LOGKEY_TRAP, postMatchBinding.spinnerTrap.getSelectedItem().toString());
-                // TODO : need to know how to build a multi-selected list of IDs (delimiter will be ":")
-//                Globals.EventLogger.LogData(Constants.LOGKEY_COMMENTS, postMatchBinding.dropComments.toString());
+                Globals.EventLogger.LogData(Constants.LOGKEY_CLIMB_POSITION, postMatchBinding.spinnerClimbPosition.getSelectedItem().toString());
+                Globals.EventLogger.LogData(Constants.LOGKEY_TRAP, postMatchBinding.spinnerTrap.getSelectedItem().toString());
+                // TODO need to know how to build a multi-selected list of IDs (delimiter will be ":")
+                Globals.EventLogger.LogData(Constants.LOGKEY_COMMENTS, postMatchBinding.dropComments.toString());
```
Comment (Jeff-Stammler) [inline app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java:208]: This isn't right.  We don't want to log the comment strings.  We need to convert this to a colon separated list of comment ID's.  Also, remove the TODO comment above.

**Review by Jeff-Stammler — CHANGES_REQUESTED** [review]
spinners look ok, but the multi-select comments isn't right.
Did you test this as well and look at the logged .csv file?

File: app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java (hunk 1 above)
Comment (AdrielStammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java:208]: Is that necessary for tonight?

Comment (Jeff-Stammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java:208]: yes.  we have to have the logging files working so we can collect the files we generate tonight to work on Tableau flows and stuff next week.

Comment (AdrielStammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java:208]: OK, I'm working on it.

Comment (Jeff-Stammler) [reply app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java:208]: Spinners also need to log the ID of the selected item, not the string.

**Review by Jeff-Stammler — APPROVED** [review]

**Pull request merged**